MAINTAINERS.md

```diff
@@ -22,11 +22,11 @@ Past CRAM maintainers include Vadim Zalunin.
 
 ### VCF/BCF
 
-* Cristina Yenyxe Gonzalez Garcia (@cyenyxe)
+* Jose Miguel Mut Lopez (@jmmut)
 * Louis Bergelson (@lbergelson)
 * Petr Danecek (@pd3)
 
-Past VCF/BCF maintainers include Ryan Poplin and David Roazen.
+Past VCF/BCF maintainers include Ryan Poplin, David Roazen and Cristina Yenyxe Gonzalez Garcia.
```

Comment:
```suggestion
Past VCF/BCF maintainers include Ryan Poplin, David Roazen, and Cristina Yenyxe Gonzalez Garcia.
```

For the record, does @cyenyxe sort herself (by surname) at G or Y?

Reply:
I was not aware of any name ordering in this page, should I change this?

Sorting by surname 'Gonzalez' should be used.

Reply:
Whether the current maintainers are sorted, and if so how, is an open question (cf #415). (If it were me, I would add myself at the bottom, but that's just MHO.)

The lists of past maintainers are **alphabetical** (and use Oxford commas).